### PR TITLE
Remove md5 dependency

### LINKED ISSUE #22: Look into letting webpack handle the hash

### PR DESCRIPTION
**Pull request by sirreal**
Remove md5 dependency.
Replace with `webpack.util.createHash`.

Closes #22

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3105,6 +3133,16 @@ prettier@^2.3.0:
   resolved "https://registry.yarnpkg.com/prettier/-/prettier-2.3.2.tgz#ef280a05ec253712e486233db5c6f23441e7342d"
   integrity sha512-lnJzDfJ66zkMy58OL5/NY5zp70S7Nz6KqcKkXYzn2tMVrNxvbqaBpg7H3qHaLxCJ5lNMsGuM8+ohS7cZrthdLQ==
 
+pretty-format@^26.0.0, pretty-format@^26.6.2:
+  version "26.6.2"
+  resolved "https://registry.yarnpkg.com/pretty-format/-/pretty-format-26.6.2.tgz#e35c2705f14cb7fe2fe94fa078345b444120fc93"
+  integrity sha512-7AeGuCYNGmycyQbCqd/3PWH4eOoX/OiCa0uphp57NVTeAGdJGaAliecxwBDHYQCIvrW7aDBZCYeNTP/WX69mkg==
+  dependencies:
+    "@jest/types" "^26.6.2"
```
Comment (blowery) [inline yarn.lock:3141]: 😿 😭

**Comment by blowery** [comment]
does this also add types for jest@26?

File: yarn.lock (hunk 1 above)
Comment (sirreal) [reply yarn.lock:3141]: This is weird. They should all be devDeps though, so 🤷

**Comment by sirreal** [comment]
Yes, it adds types for `jest`. Jest types are in a confusing place, I believe there are some first-party types but the third-party DefinitelyTyped types are more complete (this may be inaccurate). The types are likely compatible with versions 26 and 27.

**Review by blowery — APPROVED** [review]
lgtm

**Pull request merged**